Demo/Demo/SwiftUIComponents/CardVaultViews/CardVaultView.swift

```diff
@@ -3,6 +3,7 @@ import SwiftUI
 struct CardVaultView: View {
 
     @StateObject var cardVaultViewModel = CardVaultViewModel()
+    @State var sca: String? = "SCA_WHEN_REQUIRED"
```

Comment:
Since we are assigning a value from the beginning, the variable does not need to be optional, what do you think?

Reply:
I thought it might be clearer to have on top, incase default values change. Easier to find.

Reply:
It looks like this value is already the default in `CreateSetupTokenView`. We shouldn't also need to pass it at all as a default here since it's just redundant, right?

Reply:
If I don't set default here, paymentSourceType with card will be initialized with verification field nil (until picker is toggled) which omits the verification_method in the setup token create call.
Docs say SCA_WHEN_REQUIRED is the default, which I take to mean that in its absence, that is the value.

But that assignment of sca on child view right now is not automatically initializing the paymentType's verification_method property with SCA_WHEN_REQUIRED. I like the idea of initializing the default value from parent view since parent view can decide that this is PayPal payment type also.